crates/events/src/enclave_event/committee_published.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+//
+// This file is provided WITHOUT ANY WARRANTY;
+// without even the implied warranty of MERCHANTABILITY
+// or FITNESS FOR A PARTICULAR PURPOSE.
+
+use crate::E3id;
+use actix::Message;
+use serde::{Deserialize, Serialize};
+use std::fmt::{self, Display};
+
+#[derive(Message, Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[rtype(result = "()")]
+pub struct CommitteePublished {
+    pub e3_id: E3id,
+    pub nodes: Vec<String>,
+    pub public_key: Vec<u8>,
```

Comment:
Maybe this should this be `ArcBytes`? That might mean we end up being more memory efficient and can just use the Debug output in the Display formatter like this:

```rust
impl Display for CommitteePublished {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        write!(f, "{:?}", self)
    }
}
```

Without worrying about a flood of data